apps/storybook-react/stories/Card.stories.tsx

```diff
@@ -0,0 +1,347 @@
+import React from 'react'
+// import { withKnobs, select, boolean } from '@storybook/addon-knobs'
```

Comment:
Suggest we just remove this if we aren't using. Going to be replace by `controls` anyways #757